### ISSUE TITLE
bug: ng-bind-html only renders plain text of a list

### ISSUE THREAD

**Issue by petrosss** [driftyco/ionic#9543]
**Type**: <span ionic-type>bug</span>

**Ionic Version**: <span ionic-version>1.x</span>

**Platform**: <span ionic-platform>desktop</span>  <span ionic-webview>browser</span>

<span ionic-description>After a lot of searching and trying the same html in an angular application that worked fine (Angular and Angular-Sanitize version 1.5.3), the only explanation is that it must be something wrong in the way Ionic (version 1.3.1) deals with injecting HTML. Can you please let me know if you get the same problem and a possible resolution?

Thank you
Petros Papadopoulos</span>

<span is-issue-template></span>

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us! Would you mind editing your issue to fill out the issue template?

**Comment by jgw96** [comment]
This issue was moved to driftyco/ionic-v1#52

**Comment by petrosss** [comment]
After much investigation I have found what the problem was with bullet point lists not properly displaying. I was actually wrong to imply that the problem was with ng-bind-html, it will not remove bullet points from html, it will only remove some possibly harmful code. The problem is with the ionic _reset.scss file located in www\lib\ionic\scss folder that cancels some of features in order to achieve better interoperability between browsers. To solve the problem I removed the rules that blocked the display of bullet points.

Petros
